src/PyCall.jl

```diff
@@ -708,6 +708,14 @@ for (mime, method) in ((MIME"text/html", "_repr_html_"),
 end
 
 #########################################################################
+# Expose Python docstrings to Julia doc system
+
+if isdefined(Docs, :getdoc)  # Introduced in Julia .6
+    Docs.getdoc(o::PyObject) = Markdown.parse(o[:__doc__])
```

Comment:
Hmm, at least some are (https://www.tensorflow.org/versions/master/how_tos/documentation/), and the Julia markdown parser gave decent for Pandas and Seaborn. 
Should we bundle/reimplement something like doc2md?